pkg/reconciler/cluster/controller.go

```diff
@@ -280,15 +317,24 @@ func (c *Controller) deletedCluster(obj interface{}) {
 func RegisterCRDs(cfg *rest.Config) error {
 	crdClient := apiextensionsv1client.NewForConfigOrDie(cfg)
 
-	files, err := ioutil.ReadDir("config/")
+	ex, err := os.Executable()
+	if err != nil {
+		return err
+	}
+	if path.Base(ex) == "__debug_bin" {
+		ex = path.Dir(path.Dir(ex))
+	}
+
+	configDir := path.Join(path.Dir(ex), "config")
+	files, err := ioutil.ReadDir(configDir)
```

Comment:
Can you add a comment describing what this is doing? It seems like this will only work if the binary is being `go run`?

If we want to embed all these yaml files in here, we can either use `//go:embed` or take advantage of [`ko`'s build-time resource injection](https://github.com/google/ko#static-assets), which will make the files available at `$KO_DATA_PATH`.

Reply:
Yeah; it was just a way to make this a bit more robust than what was previously (it used to only work when the current directory was the KCP root directory).
In fact I added this to debug my PR main implementation more easily :-)
But you're right, so I'll remove it from the current PR, and would create a dedicated PR with a clean implementation based on `go:embed`

Reply:
Removed from this PR